composer.json

```diff
@@ -47,7 +47,7 @@
         "kartik-v/yii2-widget-select2": "^2.1",
         "kodus/file-cache": "^1.1",
         "kartik-v/yii2-widget-typeahead": "^1.0",
-        "league/iso3166": "^2.1",
+        "league/iso3166": "^4.0",
```

Comment:
Why is this upgraded? This seems be unrelated to psalm.

Reply:
composer requires this to be updates.